.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # This should match with the users in MAINTAINERS.md file
-*   @Bukhtawar @bharath-techie @Arpit-Bandejiya @dhruv16dhr @pranikum
\ No newline at end of file
+*   @Bukhtawar @bharath-techie @Arpit-Bandejiya @dhruv16dhr @pranikum @mch2 @anddross @Poojita-Raj @kotwanikunal @Rishikesh1159
```

Comment:
Should be "andrross" and not "anddross". Remove "dhruv16dhr" as the user is not a maintainer.